packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.js

```diff
@@ -22,11 +22,19 @@ import React from 'react';
 import { cssModules } from 'bpk-react-utils';
 import BpkText from 'bpk-component-text';
 import BpkButton from 'bpk-component-button';
-import BpkCard from 'bpk-component-card';
 
-import STYLES from './BpkGraphicPromo.module.scss';
+import GP_STYLES from './BpkGraphicPromo.module.scss';
```

Comment:
Can this be reverted back to `STYLES` to follow the format back as other components?

```suggestion
import STYLES from './BpkGraphicPromo.module.scss';
```